stable_baselines/acktr/acktr.py

```diff
@@ -115,9 +102,26 @@ def __init__(self, policy, env, gamma=0.99, nprocs=None, n_steps=20, ent_coef=0.
         self.trained = False
         self.continuous_actions = False
 
+        super(ACKTR, self).__init__(policy=policy, env=env, verbose=verbose, requires_vec_env=True,
+                                    _init_setup_model=_init_setup_model, policy_kwargs=policy_kwargs,
+                                    seed=seed, n_cpu_tf_sess=n_cpu_tf_sess)
+
+        if nprocs is not None:
```

Comment:
ibid: why are we moving input validation later?

Reply:
Addressed here: https://github.com/hill-a/stable-baselines/pull/639#discussion_r366134645